### PR TITLE
Origin/profile menu

### PR DESCRIPTION
**Pull request by jhubbard2017**
Main profile route added to navigation menu

### CODE REVIEW

File: src/app/routes/profile/containers/profileContainer.js [hunk 1]
```diff
@@ -14,6 +14,24 @@ export default class ProfileContainer extends React.Component {
     constructor() {
         super();
         this.editProfileAction = this.editProfileClicked.bind(this);
```
Comment (lmartin73) [inline src/app/routes/profile/containers/profileContainer.js:16]: Don't check the temporary data into master

File: trackit.iml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lmartin73) [inline trackit.iml]: Don't check this file in.

**Review by lmartin73 — CHANGES_REQUESTED** [review]

File: trackit.iml (hunk 2 above)
Comment (jhubbard2017) [reply trackit.iml]: Done

File: src/app/routes/profile/containers/profileContainer.js (hunk 1 above)
Comment (jhubbard2017) [reply src/app/routes/profile/containers/profileContainer.js:16]: Done

File: trackit.iml (hunk 2 above)
Comment (lmartin73) [reply trackit.iml]: don't check in this file.

**Review by lmartin73 — CHANGES_REQUESTED** [review]

**Review by lmartin73 — APPROVED** [review]

**Pull request merged**